Mathlib/CategoryTheory/Filtered/Flat.lean

```diff
@@ -0,0 +1,38 @@
+/-
+Copyright (c) 2025 Jakob von Raumer. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jakob von Raumer
+-/
+import Mathlib.CategoryTheory.Filtered.CostructuredArrow
+import Mathlib.CategoryTheory.Filtered.Connected
+import Mathlib.CategoryTheory.Functor.Flat
+
+/-!
+# Transfering filteredness along representably flat functors
+
+We show that if `F : C ⥤ D` is a representably flat functor between two small categories,
+filteredness of `C` implies filtereness of `D`. Dually, if `F` is representably coflat,
+cofilteredness of `D` implies cofilteredness of `C`.
+-/
+
+universe u
+
+namespace CategoryTheory
+
+open Limits
+
+variable {C : Type u} [SmallCategory C]
+variable {D : Type u} [SmallCategory D]
+variable (F : C ⥤ D)
```

Comment:
During #19684 I had missed how restricted the universes were there. It looks to me that it should be possible (if a little annoying) to recover the universe-polymorphic version of `isFiltered_of_isFiltered_costructuredArrow` through the usual dance of inserting `AsSmall.equiv` everywhere. Could you give this a try? I think it would be valuable to have a universe-polymorphic version of `isFiltered_of_representablyFlat`.